packages/fast-web-utilities/src/dom.ts

```diff
@@ -1,6 +1,31 @@
 import { canUseDOM } from "exenv-es6";
 import { isBoolean } from "lodash-es";
 
+/**
+ * A test that ensures that all arguments are HTML Elements
+ */
+export function isHTMLElement(...args: HTMLElement[]): boolean {
+    return args.every((arg: HTMLElement) => arg instanceof HTMLElement);
+}
+
+/**
+ * Returns all displayed elements inside of a root node that match a provided selector
+ */
+export function getDisplayedNodes(
+    rootNode: HTMLElement,
+    selector: string
+): HTMLElement[] {
+    if (!isHTMLElement(rootNode)) {
+        return;
+    }
+
+    const nodes: HTMLElement[] = Array.from(rootNode.querySelectorAll(selector));
+
+    // offsetParent will be null if the element isn't currently displayed,
+    // so this will allow us to operate only on visible nodes
+    return nodes.filter((node: HTMLElement) => node.offsetParent !== null);
```

Comment:
is there a potential issue if the parent of a parent is causing this to be not visible?